prototype/style.css

```diff
@@ -368,19 +368,38 @@ html, body, body > div {
 .search-input-container {
     background: #f7f5f3;
     padding: 5px 10px;
+    display: flex;
+    align-items: center;
+    flex-wrap: nowrap;
+    -webkit-flex-direction: row;
+	  flex-direction: row;
```

Comment:
not strictly necessary (default = row) but no biggie. https://developer.mozilla.org/en/docs/Web/CSS/flex-direction